stock_split_picking/README.rst

```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Split picking
+-------------
+
+This module adds a "Split" button on the outgoing pickings form.
+
+It works like the classical picking Transfer but it leaves both
+pickings (picking and it's backorder) as confirmed without
+processing the transfer.
+
+Installation
+============
+
+This module only needs `stock` module.
+
+Known issues / Roadmap
+======================
+
+ * When splitting a picking in an unassigned state, wizard won't
+   be auto completed with picking lines.
```

Comment:
It could be an uncomfortable limitation, because I guess that most of the time we split a picking because lines are not available.